remoteappmanager/static/js/home/controller.js

```diff
@@ -1,125 +1,27 @@
 /*globals: require, console*/
 require([
-    "home/init",
-    "jquery",
     "urlutils",
-    "dialogs",
-    "gamodule",
     "home/models",
     "home/views/application_list_view",
-    "home/views/application_view",
-    "jsapi/v1/resources"
+    "home/views/application_view"
 ], function(
-    init,
-    $,
     urlutils,
-    dialogs,
-    gamodule,
     models,
     application_list_view,
-    application_view,
-    resources) {
+    application_view) {
     "use strict";
 
-    var ga = gamodule.init();
```

Comment:
you can extract a separate view specifically for ga events, and respond to model changes by sending ga events.